### PR TITLE
[guides] change name from master to network or main

### PR DESCRIPTION
**Pull request by gavinmcgovern**
Fix a few more remaining instances.

### CODE REVIEW

File: site/en/guides/build/features/jam-detection.md [hunk 1]
```diff
@@ -398,7 +398,7 @@ JamDetection:RssiThreshold = -45
 ```
 
 For more information on `wpantund` properties, see the [`wpantund` GitHub
-repository](https://github.com/openthread/wpantund/tree/master/src/wpantund/wpan-properties.h).
+repository](https://github.com/openthread/wpantund/tree/main/src/wpantund/wpan-properties.h).
```
Comment (Vyrastas) [inline site/en/guides/build/features/jam-detection.md:401]: Actually, it looks like wpantund hasn't been switched from master to main (link doesn't work now), so let's keep this as master.  So on DevSite it should be "master" for the wpantund repo.

File: site/en/guides/build/logs.md [hunk 2]
```diff
@@ -249,7 +249,7 @@ Log levels may be changed at run time if dynamic log level control is enabled.
     1.  For a [system-on-chip (SoC)](https://openthread.io/platforms#single-chip-thread-only-soc),
         use the [Logging API](https://openthread.io/reference/group/api-logging) within your
         OpenThread application.
-    1.  For an NCP, use `wpanctl` on the command line. See [`wpan-properties.h`](https://github.com/openthread/wpantund/tree/master/src/wpantund/wpan-properties.h) in the `wpantund` repository for all the properties exposed to `wpanctl` and the  [Spinel API](https://github.com/openthread/openthread/blob/da12dca4d9e82cda08aba272567affa188bf8b12/src/ncp/spinel.h#L308) for its log level definitions.
+    1.  For an NCP, use `wpanctl` on the command line. See [`wpan-properties.h`](https://github.com/openthread/wpantund/tree/main/src/wpantund/wpan-properties.h) in the `wpantund` repository for all the properties exposed to `wpanctl` and the  [Spinel API](https://github.com/openthread/openthread/blob/da12dca4d9e82cda08aba272567affa188bf8b12/src/ncp/spinel.h#L308) for its log level definitions.
```
Comment (Vyrastas) [inline site/en/guides/build/logs.md:252]: Switch back to master for wpantund

**Review by Vyrastas — CHANGES_REQUESTED** [review]

**Review by gavinmcgovern — COMMENTED** [review]
Done.

File: site/en/guides/build/features/jam-detection.md (hunk 1 above)
Comment (gavinmcgovern) [reply site/en/guides/build/features/jam-detection.md]: Done.

**Review by Vyrastas — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**